hack/generated/controllers/crd_resourcegroup_test.go

```diff
@@ -12,17 +12,22 @@ import (
 	. "github.com/onsi/gomega"
 
 	"github.com/Azure/k8s-infra/hack/generated/pkg/armclient"
+	util "github.com/Azure/k8s-infra/hack/generated/pkg/util"
 )
 
 func Test_ResourceGroup_CRUD(t *testing.T) {
 	t.Parallel()
 
 	g := NewGomegaWithT(t)
+	log := util.NewBannerLogger()
+	log.Header(t.Name())
+
 	ctx := context.Background()
 	testContext, err := testContext.ForTest(t)
 	g.Expect(err).ToNot(HaveOccurred())
 
 	// Create a resource group
+	log.Subheader("Create resource group")
```

Comment:
I'm somewhat concerned about needing to remember to add these logs for each interesting place in a test. It feels like it adds difficulty in authoring a test and means I can get it "wrong" (but my test still passes).

I'm wondering if a better pattern would be to hook `testContext.KubeClient.Create` (and `Delete` and possibly `Update` too) to do this automatically by inspecting the type in question. So you'd get something like:
`"Creating %q", obj.Kind`.